crates/frontend/src/types.rs

```diff
@@ -122,10 +126,71 @@ pub struct ExperimentsResponse {
 pub struct StatusTypes(pub Vec<ExperimentStatusType>);
 
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
-pub struct ExpListFilters {
+pub struct ExperimentListFilters {
```

Comment:
is this type not there in `experimentation_platform` for the api type ?

Reply:
It is present, do you recommend moving this to superposition_types?

Reply:
its good to have, or we can wait till this PR goes in:
https://github.com/juspay/superposition/pull/291

after this PR is merged, I would raise a separate PR for moving the api types to `superposition_types` probably we can move it then